specification/content.md

```diff
@@ -43,13 +43,15 @@ content:
       ...
 ```
 
-## The Schema Object
+## The Schema Field
 
-The [Schema Object](https://spec.openapis.org/oas/v3.1.0#schema-object) defines a data type which can be a primitive (integer, string, ...), an array or an object depending on its `type` field.
+The schema field holds a [Schema Object](https://spec.openapis.org/oas/v3.1.0#schema-object).
 
-`type` is a string and its possible values are: `number`, `string`, `boolean`, `array` and `object`. Depending on the selected type a number of other fields are available to further specify the data format.
+Schema objects describe the structure of data, and may be nested to describe complex arrays and objects. They are most often used to describe JSON data.
 
-For example, for `string` types the length of the string can be limited with `minLength` and `maxLength`. Similarly, `integer` types, accept `minimum` and `maximum` values. No matter the type, if the amount of options for the data is limited to a certain set, it can be specified with the `enum` array. All these properties are listed in the [Schema Object](https://spec.openapis.org/oas/v3.1.0#schema-object) specification.
+Each schema object has a field called `type`, which defines the type of data expected. Six of the seven possible values for `type` correspond directly to JSON's types as defined in [RFC 8259](https://datatracker.ietf.org/doc/html/rfc8259): `string`, `number`, `boolean`, `null`, `object`, and `array`. However, OpenAPI defines an additional type called `integer`, which indicates a JSON number without a fraction or exponent part. In other words, if your schema object is of type `integer`, it tells your users to expect a JSON number that looks like `123`, `-123`, or `0`, but not `1.0` or `1.0e-2`.
```

Comment:
`integer` is defined by JSON Schema, not OpenAPI (we should ignore any inadvertent discrepancies between the two).  Even in OAS 3.0, `integer` as a type comes from JSON Schema even if the wording suggests otherwise.

I'm also hesitant to get into the exact definition of `integer` as it changed in subtle ways between certain JSON Schema drafts.  It's better to just let folks look at JSON Schema-related docs for that.  For example, `1.0` ***is*** an integer in the JSON Schema drafts used for 3.1 and (Im 99% sure) 3.0.  But (I think?) not in 2.0.  I'd just note that JSON Schema adds an `integer` type to its `type` keyword for convenience and leave it at that.